router/swap_inner.gno

```diff
@@ -118,3 +121,33 @@ func i256Max(x, y *i256.Int) *i256.Int {
 	}
 	return y
 }
+
+func getMinTick(fee uint32) int32 {
+	switch fee {
+	case 100:
+		return -887272
+	case 500:
+		return -887270
+	case 3000:
+		return -887220
+	case 10000:
+		return -887200
+	default:
+		panic(ufmt.Sprintf("[ROUTER] router.gno__getMinTick() || unknown fee(%d)", fee))
+	}
+}
+
+func getMaxTick(fee uint32) int32 {
+	switch fee {
+	case 100:
+		return 887272
+	case 500:
+		return 887270
+	case 3000:
+		return 887220
+	case 10000:
+		return 887200
+	default:
+		panic(ufmt.Sprintf("[ROUTER] router.gno__getMaxTick() || unknown fee(%d)", fee))
+	}
+}
```

Comment:
`gno.land/r/gnoswap/consts` It looks like MAX_TICK is declared in the package here.
What do you think about managing the value of MIN / MAX TICK per fee tier ?